src/app/vm-logs/services/vm-log-files.service.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { BackendResource } from '../../shared/decorators';
+import { BaseBackendService, FormattedResponse } from '../../shared/services/base-backend.service';
+import { VmLogFile } from '../models/vm-log-file.model';
+
+
+@Injectable()
+@BackendResource({
+  entity: 'VmLogFile'
+})
+export class VmLogFilesService extends BaseBackendService<VmLogFile> {
+  constructor(protected http: HttpClient) {
+    super(http);
+  }
+
+  protected formatGetListResponse(response: any): FormattedResponse<VmLogFile> {
+    const result = response && response.vmlogfiles || [];
+    return {
+      list: result.map(m => this.prepareModel(m)) as Array<VmLogFile>,
```

Comment:
fetch last changes
we dont have method prepareModel in baseBackendService anymore

Reply:
I will update to master when merging https://github.com/bwsw/cloudstack-ui/pull/1344